site/src/components/AvatarData/AvatarData.tsx

```diff
@@ -1,71 +1,50 @@
-import Avatar from "@material-ui/core/Avatar"
-import Link from "@material-ui/core/Link"
-import { makeStyles } from "@material-ui/core/styles"
+import { Avatar } from "components/Avatar/Avatar"
 import { FC, PropsWithChildren } from "react"
-import { Link as RouterLink } from "react-router-dom"
-import { firstLetter } from "../../util/firstLetter"
-import {
-  TableCellData,
-  TableCellDataPrimary,
-  TableCellDataSecondary,
-} from "../TableCellData/TableCellData"
+import { Stack } from "components/Stack/Stack"
+import { makeStyles } from "@material-ui/core/styles"
 
 export interface AvatarDataProps {
   title: string
   subtitle?: string
-  highlightTitle?: boolean
-  link?: string
+  src?: string
   avatar?: React.ReactNode
```

Comment:
This is not really important, just a thought but would it make sense to make `AvatarData` take `Avatar` props and pass them along rather than accept `avatar`?  My reasoning is that it seems unfortunate that there are two ways of doing things like:

```
<AvatarData src={src} />
```
and

```
<AvatarData avatar={<Avatar src={src} />} />
```

Or alternatively we could always make it so we always have to pass an `avatar`, either way works so there is one way of doing things.

Reply:
`AvatarData` is a bad name for what it does and not a good abstraction. Wondering if `Heading` or `DataHeading` would be better with the following interface:

```tsx
<Heading
  title="Bruno"
  subtitle="bruno@hotmail.com"
  leftElement={<UserAvatar user="{user}" />}
/>
```

Reply:
Definitely agree!